### PR TITLE
Fix generated validation error message for wrong exception mapping status code

### PR DESCRIPTION
**Pull request by andrew-demb**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no 
| Deprecations? | no 
| Tickets       |  
| License       | MIT
| Doc PR        |  

There was a typo in validation error.

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -1235,7 +1235,7 @@ private function addExceptionsSection(ArrayNodeDefinition $rootNode)
                                 ->info('The status code of the response. Null to let Symfony decide.')
```
Comment (andrew-demb) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php:1235]: Additionally, there is no possibility to explicitly specify `null` due to the validation (info message can be misleading).

Would you accept a fix for validation to allow this case?

**Comment by alexislefebvre** [comment]
I think the PR should target the 5.4 branch: https://symfony.com/doc/current/contributing/code/pull_requests.html#choose-the-right-branch

**Comment by andrew-demb** [comment]
CI failure is unrelated

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (chalasr) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php:1235]: Indeed good catch, PR welcome.

**Review by chalasr — APPROVED** [review]

**Comment by chalasr** [comment]
Good catch, thanks @andrew-demb.

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (andrew-demb) [reply src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php:1235]: Here it is https://github.com/symfony/symfony/pull/46956

**Pull request merged**